classes/class.ilSelfEvaluationPlugin.php

```diff
@@ -35,4 +35,38 @@ public function allowCopy(): bool
     {
         return true;
     }
+
+    /**
+     * @description This is the easiest way to fix all locations which use a template.
+     * And this method most likely will work in ILIAS 9 and 10.
```

Comment:
this was the ILIAS 10 way, there is another commit which changes this slightly to work in ILIAS 9. I didn't find a way to have the same code for 9 and 10.